src/ImpulseKelvinlet.cpp

```diff
@@ -24,15 +24,15 @@ glm::vec3 ImpulseKelvinlet::displacement(glm::vec3 position, Material material,
     return -3.0f * pow(scale, 4) * (pow(reg_s, 2) - 5 * r * s) / pow(reg_s, 7);
   };
 
-  auto full_W = [&](double s) -> glm::vec3 {
+  auto full_W = [&](double s) -> glm::vec2 {
     double reg_s = reg(s);
-    return {W(s, reg_s), dr_W(s, reg_s), drr_W(s, reg_s)};
+    return {W(s, reg_s), dr_W(s, reg_s)};
```

Comment:
I noticed that push Kelvinlets actually never use the second derivative of W here so I'm just taking it out

Reply:
sounds good.